<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTabWidget</class>
 <widget class="QWidget" name="MapTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,0" columnstretch="0,10,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="2">
    <widget class="QDockWidget" name="spipDD2DockWidget">
     <widget class="NiippWidget" name="spipDD2Widget"/>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDockWidget" name="bplaDockWidget">
     <property name="windowTitle">
      <string>BPLA List</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeView" name="bplaTreeView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDockWidget" name="spipDD1DockWidget">
     <widget class="NiippWidget" name="spipDD1Widget"/>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="MapWidget" name="mapWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDockWidget" name="blaDockWidget">
     <property name="windowTitle">
      <string>BLA List</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeView" name="blaTreeView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>MapInterface/MapWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NiippWidget</class>
   <extends>QWidget</extends>
   <header>Niipp/NiippWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
